milestones/models.py

```diff
@@ -0,0 +1,113 @@
+# pylint: disable=no-init
+# pylint: disable=old-style-class
+# pylint: disable=too-few-public-methods
+"""
+Database ORM models managed by this Django app
+Please do not integrate directly with these models!!!  This app currently
+offers two APIs -- api.py for direct Python integration and receivers.py,
+which leverages Django's signal framework.
+"""
+
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+
+class Milestone(TimeStampedModel):
+    """
+    A Milestone is a representation of an accomplishment which can be
+    attained by a user. Milestones have a base set of meta data
+    describing the milestone, including id, name, and description.
+    Milestones can be used to drive functionality and behavior 'behind
+    the scenes' in Open edX, such as with the Pre-Requisite Course and
+    Course Entrance Exam use cases.
+    """
+    namespace = models.CharField(max_length=255)
+    name = models.CharField(max_length=255)
```

Comment:
sorry, one more comment: should we have a "display_name". "name" here is more like an internal system name, correct? I could imagine that we'd at some point like to render a list of milestones for a user and having a display_name would be handy.